django-stubs/db/models/enums.pyi

```diff
@@ -34,5 +40,5 @@ class _TextChoicesMeta(ChoicesMeta):
     values: List[str] = ...
 
 class TextChoices(str, Choices, metaclass=_TextChoicesMeta):
-    @property
+    @enum_property
```

Comment:
This won't be supported by `mypy`.

Reply:
Yes I think this is necessary. The same errors also appear in https://github.com/typeddjango/django-stubs/pull/1223

Maybe add a test with `reveal_type()` on the `value` property just in case?

Reply:
> The same errors also appear in #1223

That is this PR, did you mean to link to a different one?

Reply:
Ah yes, I meant https://github.com/typeddjango/djangorestframework-stubs/pull/284